stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGalleryActivity.kt

```diff
@@ -144,5 +176,29 @@ public class AttachmentGalleryActivity : AppCompatActivity() {
         val image: String? = null,
         val url: String? = null,
         val name: String? = null,
-    ) : Parcelable
+    ) : Parcelable {
+        public fun toAttachment(): Attachment =
+            Attachment(
+                authorName = authorName,
+                imageUrl = imageUrl,
+                assetUrl = assetUrl,
+                url = url,
+                name = name,
+                image = image,
+                type = type,
+                text = text,
+                title = title,
+                fileSize = fileSize,
+                mimeType = mimeType,
+            )
+    }
+
+    public interface AttachmentOptionHandler {
+        public fun onClick(attachmentData: AttachmentData): Unit
+    }
```

Comment:
We don't use this type anywhere other than the declarations of the other interfaces. This way it could be implemented / accidentally used. Let's just duplicate that single function declaration in the other interfaces.